### DISCUSSION TITLE
What is the latest version for podman and podman-remote for redhat9?

### DISCUSSION THREAD

**Discussion by jumpingliu** [containers/podman#22692] [Q&A] [answered]
My OS is redhat9 and recently I want to upgrade the podman version to the latest one, but when I install podman using the following command line, it only gets the 4.9.4, not sure if 4.9.4 is the latest version.
 
`sudo yum install -y podman`
`sudo yum install -y podman-remote`

My current version is `podman version 4.9.4-rhel` and podman-remote version 4.9.4-rhel

here says 5.0.3 is out now: https://github.com/containers/podman/releases

Thanks in advance:)

**Comment by vrothberg** [comment, marked as answer]
Thanks for reaching out, @jumpingliu.  Podman 4.9.4 is the latest version in RHEL.  It takes a while until the latest upstream versions go into the downstream distros such as RHEL; they're usually shipped with a new version of RHEL.

**Reply by jumpingliu** [reply]
@vrothberg Thanks a lot for the response and the explanation!!!(Took me this morning to dig into it 😂)

**Reply by tcortega** [reply]
@vrothberg Version hasn't changed since may, is that still the case? If yes why does RHEL doesn't have the latest version of podman?

**Reply by rhatdan** [reply]
RHEL updates to the latest version each 6 months.  Podman 5.2 or later will be in RHEL 9.6

**Reply by tcortega** [reply]
> RHEL updates to the latest version each 6 months. Podman 5.2 or later will be in RHEL 9.6

@rhatdan I'm a bit confused on how that works, couldn't I simply add a repo and install Podman 5.2? Or does podman not support it yet.

**Reply by rhatdan** [reply]
Theoretically you could build your own podman or grab the packages from centos-stream.  But that would not be supported.  Podman requires a whole bunch of other packages container-selinux, crun, conmon, netavark, aardvar-dns, containers-common off the top of my head.
